rusoto/services/sagemaker-runtime/src/generated.rs

```diff
@@ -112,17 +112,17 @@ impl InvokeEndpointError {
 
             match error_type {
                 "InternalFailure" => {
-                    return InvokeEndpointError::InternalFailure(String::from(error_message))
+                    return InvokeEndpointError::InternalFailure(String::from(error_message));
                 }
                 "ModelError" => return InvokeEndpointError::ModelError(String::from(error_message)),
                 "ServiceUnavailable" => {
-                    return InvokeEndpointError::ServiceUnavailable(String::from(error_message))
+                    return InvokeEndpointError::ServiceUnavailable(String::from(error_message));
                 }
                 "ValidationError" => {
-                    return InvokeEndpointError::ValidationError(String::from(error_message))
+                    return InvokeEndpointError::ValidationError(String::from(error_message));
                 }
                 "ValidationException" => {
-                    return InvokeEndpointError::Validation(error_message.to_string())
+                    return InvokeEndpointError::Validation(error_message.to_string());
```

Comment:
The changes above could be caused by a nightly version of rust which I used for generation process. They look reasonable, so I'm not sure if it's necessary to re-generate.

Reply:
We're currently using the `stable` version of `rustfmt`. The SemaphoreCI build verifies it can regenerate the code in the PR with this configuration, which is why that build failed.

I'd love to make this more clear so nobody else trips over it. Any ideas on how? 👍

Reply:
My bad, I didn't figure out that "stable rustfmt" means "stable rust". I've re-generated with stable.

I saw you started Makefile. I think it's a right way to go. Another way is to explicitly specify `stable` version in example commands in README. I'll add a PR for this.

Reply:
Proposal is here:  #1262
It could be helpful until the Makefile is finished.